<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataListAddColumn</class>
 <widget class="QDialog" name="DataListAddColumn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add column</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelStation">
       <property name="text">
        <string>&amp;Station:</string>
       </property>
       <property name="buddy">
        <cstring>textStation</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="textStation"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelParam">
       <property name="text">
        <string>&amp;Parameter:</string>
       </property>
       <property name="buddy">
        <cstring>comboParam</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboParam">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>118</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelType">
       <property name="text">
        <string>&amp;Type:</string>
       </property>
       <property name="buddy">
        <cstring>comboType</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboType">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelLevel">
       <property name="text">
        <string>&amp;Level:</string>
       </property>
       <property name="buddy">
        <cstring>comboLevel</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboLevel">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelSensorNumber">
       <property name="text">
        <string>S&amp;ensor:</string>
       </property>
       <property name="buddy">
        <cstring>spinSensorNumber</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinSensorNumber">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelTimeOffset">
       <property name="text">
        <string>T&amp;ime offset:</string>
       </property>
       <property name="buddy">
        <cstring>spinTimeOffset</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinTimeOffset">
       <property name="toolTip">
        <string>Time shift for the column. A time shift of 1 hour means that data with, e.g., observation time 08:00 will be shown in the row for 09:00.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string>h</string>
       </property>
       <property name="minimum">
        <number>-48</number>
       </property>
       <property name="maximum">
        <number>48</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioCorrected">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Corrected Value</string>
       </property>
       <property name="statusTip">
        <string>Corrected Value</string>
       </property>
       <property name="text">
        <string>&amp;Corr</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioOriginal">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Original Value</string>
       </property>
       <property name="statusTip">
        <string>Original Value</string>
       </property>
       <property name="text">
        <string>&amp;Orig</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioFlags">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Flags</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioModel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Model value</string>
       </property>
       <property name="statusTip">
        <string>Model value</string>
       </property>
       <property name="text">
        <string>&amp;Model</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="2"/>
 <tabstops>
  <tabstop>textStation</tabstop>
  <tabstop>comboParam</tabstop>
  <tabstop>comboType</tabstop>
  <tabstop>radioCorrected</tabstop>
  <tabstop>radioOriginal</tabstop>
  <tabstop>radioFlags</tabstop>
  <tabstop>radioModel</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>DataListAddColumn</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>DataListAddColumn</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onStationEdited()</slot>
  <slot>onParameterSelected(int)</slot>
  <slot>onTypeSelected(int)</slot>
 </slots>
</ui>
